uktrade/data-hub-frontend
# Reactify OMIS edit internal information page

## Description of change

The form for editing the internal information for an OMIS order has been reactified. It's also now possible to save multiple service types (this was the intended functionality but when adding multiple service types in the current Nunjucks form only the first is saved).

## Test instructions

Go to an OMIS order. It should be possible to edit the internal information fields.

## Screenshots

### Before

![Screenshot 2023-09-27 at 22-15-48 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/7ebff68e-b5ef-4ac8-83d2-b2de9190cc4e)


### After

![Screenshot 2023-09-27 at 22-15-32 Edit internal information - SDE234_91 - Orders (OMIS) - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/0c8ded2d-ea5e-4f22-936d-35f483268715)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
